distribution/changelog.txt

```diff
@@ -1,5 +1,6 @@
 0.4.6 (in development)
 ------------------------------------------------------------------------
+- add: [#20587] Add Steel Spinning Roller Coaster Track
```

Comment:
Make sure to keep to the standard used everywhere else in the changelog. Don't forget about punctuation either!

Reply:
I will change this how does the standard looks like?

Reply:
Please refer to other ride type additions in the changelog. "- Feature [#20597] Add Steel Spinning Roller Coaster ride type."